archive/j/javascript/even-odd.js

```diff
@@ -0,0 +1,6 @@
+const input = Number(process.argv[2]); //coerce the input into a number if possible
+if(!Number.isInteger(input)){ //if there is no input, input = undefined and the statement still prints
+    console.log('Usage: please input an integer as the first command line argument');
```

Comment:
Looks like this is a bit more detailed than we have in testing. 
```suggestion
    console.log('Usage: please input a number');
```